docs/quickstart.md

```diff
@@ -0,0 +1,347 @@
+> Before installing the LocalPV-ZFS driver please make sure your Kubernetes Cluster meets the following prerequisites:
+> 1. All the nodes must have ZFS utils package installed
+> 2. A ZPOOL has been configurred for provisioning volumes
+> 3. You have access to install RBAC components into kube-system namespace. The OpenEBS ZFS driver components are installed in kube-system namespace to allow them to be flagged as system critical components.
+<BR>
+
+## Setup
+
+All nodes should have the same verion of zfsutils-linux installed. <BR>
+
+```bash
+$ apt-get install zfsutils-linux
+```
+
+Go to each node and create the ZFS Pool, which will be used for provisioning the volumes. You can create the Pool of your choice, it can be striped, mirrored or raidz pool.
+
+If you have the disk(say /dev/sdb) then you can use the below command to create a striped pool :
+```bash
+$ zpool create zfspv-pool /dev/sdb
+```
+You can also create mirror or raidz pool as per your need. Check https://github.com/openzfs/zfs for more information.
+
+
+If you don't have the disk, then you can create the zpool on the loopback device which is backed by a sparse file. Use this for testing purpose only.
+```bash
+$ truncate -s 100G /tmp/disk.img
+$ zpool create zfspv-pool `losetup -f /tmp/disk.img --show`
+```
+
+Once the ZFS Pool is created, verify the pool via `zpool status` command, you should see something like this :
+```bash
+$ zpool status
+  pool: zfspv-pool
+ state: ONLINE
+  scan: none requested
+config:
+
+    NAME        STATE     READ WRITE CKSUM
+    zfspv-pool  ONLINE       0     0     0
+      sdb       ONLINE       0     0     0
+
+errors: No known data errors
+```
+
+Configure the custom topology keys (if needed). This can be used for many purposes like if we want to create the PV on nodes in a particuler zone or building. We can label the nodes accordingly and use that key in the storageclass for taking the scheduling decesion:
+
+https://github.com/openebs/zfs-localpv/blob/HEAD/docs/faq.md#6-how-to-add-custom-topology-key
+
+<BR>
+
+## Installation
+In order to support moving data to a new node later on, you must label each node with a unique value for `openebs.io/nodeid`.
+For more information on migrating data, please [see here](docs/faq.md#8-how-to-migrate-pvs-to-the-new-node-in-case-old-node-is-not-accessible)
+
+**NOTE:** Installation using operator YAMLs is not the supported way any longer.  
+We can install the latest release of OpenEBS ZFS driver by running the following command:
+```bash
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
+helm install openebs --namespace openebs openebs/openebs --create-namespace
+```
+
+**NOTE:** If you are running a custom Kubelet location, or a Kubernetes distribution that uses a custom Kubelet location, the `kubelet` directory must be changed on the helm values at install-time using the flag option `--set zfs-localpv.zfsNode.kubeletDir=<your-directory-path>` in the `helm install` command.
```

Comment:
If you are running a custom Kubelet, or a Kubernetes distribution...

Reply:
`If using a custom kubelet directory, or a different flavour of Kubernetes distribution, specify it during installation:` --> Changed to this.